```
=== RUN   TestCandidateLastReceived
    /home/atoppi/src/pion/ice/candidate_test.go:196:
                Error Trace:    /home/atoppi/src/pion/ice/candidate_test.go:196
                Error:          Not equal:
                                expected: time.Date(2025, time.December, 17, 17, 41, 59, 709426140, time.Local)
                                actual  : time.Date(2025, time.December, 17, 17, 41, 59, 709426131, time.Local)

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1,2 +1,2 @@
                                -(time.Time) 2025-12-17 17:41:59.70942614 +0100 CET m=+0.000883993
                                +(time.Time) 2025-12-17 17:41:59.709426131 +0100 CET m=+0.000883993

                Test:           TestCandidateLastReceived
--- FAIL: TestCandidateLastReceived (0.00s)
FAIL
FAIL    github.com/pion/ice/v4  0.004s
```
Any idea for this randomic tiny error?

--- a/candidate_test.go
+++ b/candidate_test.go
@@ -185,7 +185,7 @@ func TestCandidateLastSent(t *testing.T) {
 	require.Equal(t, candidate.LastSent(), time.Time{})
 	now := time.Now()
 	candidate.setLastSent(now)
-	require.Equal(t, candidate.LastSent(), now)
+	require.WithinDuration(t, candidate.LastSent(), now, 1*time.Microsecond)
 }
 
 func TestCandidateLastReceived(t *testing.T) {
@@ -193,7 +193,7 @@ func TestCandidateLastReceived(t *testing.T) {
 	require.Equal(t, candidate.LastReceived(), time.Time{})
 	now := time.Now()
 	candidate.setLastReceived(now)
-	require.Equal(t, candidate.LastReceived(), now)
+	require.WithinDuration(t, candidate.LastReceived(), now, 1*time.Microsecond)
 }
 
 func TestCandidateFoundation(t *testing.T) {

